### ISSUE TITLE
Blank ElevationGraph tab

### ISSUE THREAD

**Issue by sikmir** [tumic0/GPXSee#106]
### Steps to reproduce
1. Get GPX track and open:
```sh
$ wget -qO- http://www.valentinvolk.com/Timan_2016/plan/trek.zip | zcat > trek.gpx
$ gpxsee trek.gpx
```
All track points in the file have `<ele>0</ele>`.

### Actual behaviour
Blank ElevationGraph tab:
![screenshot1](https://user-images.githubusercontent.com/688044/39158250-9bbe310e-4767-11e8-9015-b618caa0c9d8.jpg)

### Configuration
GPXSee Version: 5.9
OS: ArchLinux

**Comment by tumic0** [comment]
You are trying to display the graph with time x axis which can't work on the provided data. The GPX file has some mixture of times that do not form an ascending sequence (the first entries have a 2016 year where the later ones have 1970...). Only graphs with the distance x axis can be displayed on data with broken timestamps.

**Comment by sikmir** [comment]
Maybe it's better not to show Elevation tab at all in such case...

**Comment by tumic0** [comment]
This would evoke that there is no elevation at all, which is not true.

**Comment by sikmir** [comment]
@tumic0 One more related case, GPX file contains route (points without elevation and time) and track (points with elevation and time).
Distance X axis:
![src1](https://user-images.githubusercontent.com/688044/66681576-a8e19780-ec7b-11e9-9e86-1e1736cf60f3.jpg)
Time X axis:
![src2](https://user-images.githubusercontent.com/688044/66681577-a8e19780-ec7b-11e9-80a7-fba08effd044.jpg)
I think elevation graph should be displayed in the first case as well, right?

**Comment by tumic0** [comment]
Can you please provide the GPX file so I can have a look at it? The first image definitely looks weird...

**Comment by sikmir** [comment]
> Can you please provide the GPX file so I can have a look at it?

Sure, here it is [201905-vmb.gpx.gz](https://github.com/tumic0/GPXSee/files/3721159/201905-vmb.gpx.gz)

**Comment by tumic0** [comment]
![ss](https://user-images.githubusercontent.com/11480505/66713462-8d75aa00-edab-11e9-9c1b-1b515b69f541.png)
I can't reproduce it with the file no matter what I try... What GPXSee version do you have?

**Comment by sikmir** [comment]
> I can't reproduce it with the file no matter what I try... What GPXSee version do you have?

Oh, I see, I've used 7.13, but this problem fixed in 7.14.
